docs/UserGuide.md

```diff
@@ -223,6 +223,25 @@ You can use this command when you want to add an entry (expense/revenue) to the
 
 </div>
 
+<div markdown="block" class="alert alert-danger">
+
+:warning: **Warning:**
+
```

Comment:
I think this is more of a note instead of a warning

Reply:
haha actl i put under notes, but the doc was under warning but ok i'll change it back

Reply:
ohh okok can, warnings are for like irreversible effects, like delete account